proxylib/test/accesslog_server.go

```diff
@@ -0,0 +1,137 @@
+// Copyright 2017, 2018 Authors of Cilium
```

Comment:
I didn't notice the other files `Copyright 2017, 2018 Authors of Cilium` -> `Copyright 2017-2018 Authors of Cilium`